File: 3019-furthest-point-from-origin/furthest-point-from-origin.go
func furthestDistanceFromOrigin(moves string) int {
    l,r,n:=0,0,0
    for _,v:= range moves{
        if v=='L'{
            l++
        }else if v=='R'{
            r++
        }else{
            n++
        }
    }
    if l>r{
        return (l+n)-r
    }else if r>l{
        return (r+n)-l
    }else{
        return n
    }
}

File: 3019-furthest-point-from-origin/furthest-point-from-origin.py
class Solution:
    def furthestDistanceFromOrigin(self, moves: str) -> int:
        l=0
        r=0
        n=0
        for v in moves:
            if v=="L":
                l+=1
            elif v=="R":
                r+=1
            else:
                n+=1
        if l>r:
            return l+n-r
        elif r>l:
            return r+n-l
        else:
            return n

        